docs/assets/js/_partials/git-versions-database.js

```diff
@@ -0,0 +1,110 @@
+class GitVersionsDatabase
+{
+    // This is a singleton class to avoid loading the Git versions database multiple times
+    constructor()
+    {
+        if (GitVersionsDatabase._instance)
+            return GitVersionsDatabase._instance;
+
+        GitVersionsDatabase._instance = this;
+    }
+
+    load(callback, customDataURL = undefined)
+    {
+        // If the data has already been collected, immediately return it
+        if (this.data != undefined)
+        {
+            callback(null, this.data);
+            return;
+        }
+
+        const dataURL = (customDataURL != undefined) ? customDataURL : GitVersionsDatabase.dataURL;
```

Comment:
I wonder if we should let the Hubble user activate the call to github.com here explicitly?

Reply:
I thought of this as well. However, I think that a front-end setting would be too much work to implement at this stage (after all, this pull request already is gigantic). An easy option might be to add a configuration option to the dashboard front-end to explicitly opt out of the online updates. I’m not sure if anyone would be interested in such an option though. But I might be misjudging the situation, and maybe, some folks are legally restricted in what they are allowed to fetch from online sources within their company’s networks?

Reply:
OK. If you leave it as is, then I would make this "loads data from github.com" very explicit in the readme.md. AFAIK we don't load any other resource from the web, right?

Reply:
@larsxschneider: My suggestion would be to write a new (informational, not legally binding) disclaimer section explaining what kinds of data Hubble records from the appliance and fetches from the web. I’d open a separate pull request for that stuff, because it’s not really related to the idea of tracking vulnerable Git versions in the first place.